src/clients/shopify.js

```diff
@@ -60,7 +63,7 @@ const call = async (method, url, requestPayload) => {
  * @return The data {Object}
  */
 const getProducts = async productIds => (
-  call('GET', `/products.json?ids=${productIds.join(',')}`)
+  call(METHOD_GET, `/products.json?ids=${productIds.join(',')}`)
```

Comment:
```suggestion
  call(METHOD_GET, `/products.json?ids=${productIds.join(',')}`)
```
call(METHOD_GET, `/products.json?ids=${productIds.join()}`)